apps/src/lib/kits/maker/Accelerometer.js

```diff
@@ -4,9 +4,7 @@ import {sensor_channels, roundToHundredth} from './MicroBitConstants';
 // Transfer the acceleration units from milli-g to meters/second^2
 // Round to the nearest hundredth
 function unitsFromMGToMS2(val) {
-  // Using 1024 as a divisor because the Microbit gives values between 0 and 1024
-  // to represent the values between 0 and 1g
-  let g = val / 1024;
+  let g = val / 1000;
```

Comment:
why did this divisor change from 1024 to 1000?

Reply:
oh nvm -- just saw the comment that `val` is in milli-g, so we're just converting to g